Money/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
```

Comment:
Your CSS isn't properly linked here because you named the file style.css and this line points to styleS.css. I like what you were doing when I properly linked them.